DSharpPlus/Entities/Voice/DiscordVoiceState.cs

```diff
@@ -141,6 +142,16 @@ public DiscordUser User
         [JsonProperty("request_to_speak_timestamp", NullValueHandling = NullValueHandling.Ignore)]
         internal DateTimeOffset? RequestToSpeakTimestamp { get; set; }
 
+        /// <summary>
+        /// Gets the member this voice state belongs to.
+        /// </summary>
+        [JsonIgnore]
+        public DiscordMember Member
+            => new(this.TransportMember) { Discord = this.Discord };
```

Comment:
Is there a way for us to get this from cache first rather than always instantiating a new object every time we reference this?